src/runtime/basic-runtime.js

```diff
@@ -313,6 +314,21 @@ export class ConfiguredBasicRuntime {
         : Promise.resolve()
     );
 
+    // Start listening to Audience Activity events.
+    if (
+      isExperimentOn(
+        this.doc_.getWin(),
+        ExperimentFlags.LOGGING_AUDIENCE_ACTIVITY
+      )
+    ) {
+      /** @private @const {!AudienceActivityEventListener} */
+      this.audienceActivityEventListener_ = new AudienceActivityEventListener(
```

Comment:
Could you test this new if statement?
https://app.codecov.io/gh/subscriptions-project/swg-js/compare/2044/diff